Microsoft.DotNet.Interactive/KernelExtensionPathResolver.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using MLS.Agent.Tools;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public class KernelExtensionPathResolver
+    {
+        public static bool TryGetExtensionPath(DirectoryInfo directory, IKernel handlingKernel, out DirectoryInfo extensionDirectory)
```

Comment:
This is fairly vague contract. Should it be a relative or absolute path? How does the caller know what path to pass in?

Reply:
Modified to return a directoryaccessor